### DISCUSSION TITLE
Test target port changes result in program failure to detect results

### DISCUSSION THREAD

**Discussion by sectest2024** [projectdiscovery/nuclei#4959] [Q&A]
![image](https://github.com/projectdiscovery/nuclei/assets/47935274/bf02a169-2c6a-4dcb-bfc5-e4eb2a27d9f0)
How to make 1 and 2 in the graph consistent

**Comment by ehsandeep** [comment]
@sectest2024 are you looking create template to replicate above? or you have template that expected to follow above but not working? in either case please share template to review!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
